test/integration/registration/spokecluster_autoapproval_test.go

```diff
@@ -130,8 +130,10 @@ var _ = ginkgo.Describe("Cluster Auto Approval", func() {
 		updatedCluster := cluster.DeepCopy()
 		updatedCluster.Spec.HubAcceptsClient = false
 
-		_, err = clusterClient.ClusterV1().ManagedClusters().Update(context.TODO(), updatedCluster, v1.UpdateOptions{})
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Eventually(func() error {
+			_, err = clusterClient.ClusterV1().ManagedClusters().Update(context.TODO(), updatedCluster, v1.UpdateOptions{})
```

Comment:
I do not think it works, you need to also call get in eventually.

Reply:
You're right, updated.